Logs / errors are lost due to `wsgi.errors` not extracted

Hey there and thanks for a fancy tool!

I've noticed that interceptor puts a `io.BytestIO` to `wsgi.errors`.
That causes some trouble:
* it is never read -- so logs from my flask app crashes are lost
* flask attempts to write a `unicode` there and fails with
```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/logging/__init__.py", line 882, in emit
    stream.write(fs % msg)
TypeError: 'unicode' does not have the buffer interface
Logged from file app.py, line 1560
```

IMO putting a `sys.stderr` there would actually improve the situation.

If it is a viable option I can go ahead and produce a pull request.

Here is the MVE:
```python
import flask
import requests

from wsgi_intercept.interceptor import RequestsInterceptor


app = flask.Flask(__name__)


@app.route('/')
def foo():
    # cause a KeyError
    return flask.jsonify({}['foo'])


def test_foo():
    with RequestsInterceptor(lambda: app, host='foo.com', port=80) as url:
        r = requests.get(url)

        assert r.status_code == 200
```
Running it causes the error:
```
(py27) pupssman@pupssman-ub:~/git/wsgi_interseptor_51_mve $ py.test test.py 
===================================================================================================================================================== test session starts =====================================================================================================================================================
platform linux2 -- Python 2.7.12, pytest-2.9.0, py-1.4.34, pluggy-0.3.1
rootdir: /home/pupssman/git/wsgi_interseptor_51_mve, inifile: 
collected 1 items 

test.py F

========================================================================================================================================================== FAILURES ===========================================================================================================================================================
__________________________________________________________________________________________________________________________________________________________ test_foo ___________________________________________________________________________________________________________________________________________________________

    def test_foo():
        with RequestsInterceptor(lambda: app, host='foo.com', port=80) as url:
            r = requests.get(url)
    
>           assert r.status_code == 200
E           assert 500 == 200
E            +  where 500 = <Response [500]>.status_code

test.py:20: AssertionError
---------------------------------------------------------------------------------------------------------------------------------------------------- Captured stderr call -----------------------------------------------------------------------------------------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/logging/__init__.py", line 882, in emit
    stream.write(fs % msg)
TypeError: 'unicode' does not have the buffer interface
Logged from file app.py, line 1560
================================================================================================================================================== 1 failed in 0.12 seconds ===================================================================================================================================================
(py27) pupssman@pupssman-ub:~/git/wsgi_interseptor_51_mve $ 
```

--- a/wsgi_intercept/__init__.py
+++ b/wsgi_intercept/__init__.py
@@ -313,7 +313,7 @@ def make_environ(inp, host, port, script_name):
         "wsgi.version": (1, 0),
         "wsgi.url_scheme": "http",
         "wsgi.input": inp,  # to read for POSTs
-        "wsgi.errors": BytesIO(),
+        "wsgi.errors": sys.stderr,
         "wsgi.multithread": 0,
         "wsgi.multiprocess": 0,
         "wsgi.run_once": 0,

